kymatio/scattering2d/core/scattering2d.py

```diff
@@ -73,10 +68,15 @@ def scattering2d(x, pad, unpad, backend, J, L, phi, psi, max_order):
             S_2_r = fft(S_2_c, 'C2R', inverse=True)
             S_2_r = unpad(S_2_r)
 
-            S[..., n_order2, :, :] = S_2_r
-            n_order2 += 1
+            out_S_2.append(S_2_r)
+
+    out_S = []
+    out_S.extend(out_S_0)
+    out_S.extend(out_S_1)
+    out_S.extend(out_S_2)
```

Comment:
This passes the tests yes?

Reply:
It does locally, but it's always good to check on Jenkins/Travis.